synapsis/src/test/java/com/soen/synapsis/unit/appuser/connection/ConnectionServiceTest.java

```diff
@@ -0,0 +1,72 @@
+package com.soen.synapsis.unit.appuser.connection;
```

Comment:
Could you write tests for the other classes you created? It's part of our code coverage metrics.

Reply:
Added the tests for Connection, ConnectionKey, and ConnectionController.